rpm/SPECS/contextBroker.spec

```diff
@@ -117,8 +117,10 @@ rm $RPM_BUILD_ROOT/usr/share/contextBroker/tests/cases/CMakeLists.txt
 cp LICENSE $RPM_BUILD_ROOT/usr/share/doc/contextBroker
 cp scripts/testEnv.sh test/functionalTest/testHarness.sh test/functionalTest/testDiff.py test/functionalTest/harnessFunctions.sh $RPM_BUILD_ROOT/usr/share/contextBroker/tests
 cp scripts/accumulator-server.py $RPM_BUILD_ROOT/usr/share/contextBroker/tests 
+cp scripts/httpsPrepare.sh $RPM_BUILD_ROOT/usr/share/contextBroker/tests
```

Comment:
We have the script, but I don't think it's reachable yet.

For example, take this line in `0000_https_support/https.test`:
    
    ${SCRIPT_HOME}/../../scripts/httpsPrepare.sh --keyFileName /tmp/harnessTest.key --certFileName /tmp/harnessTest.pem

If the test is run from `/usr/share/contextBroker/tests` then this points to `/usr/scripts/httpsPrepare.sh`.